flixel/animation/FlxAnimation.hx

```diff
@@ -277,4 +277,20 @@ class FlxAnimation extends FlxBaseAnimation
 	{
 		return _frames.length;
 	}
+	
+	/**
+	 * Appends the reverse of the animation to its frames, without duplicating the original last frame.
+	 * 
+	 * If looped is true, the original first frame is also not duplicated.
+	 * 
+	 * E.g. [0,1,2,3] becomes [0,1,2,3,2,1]   (looped)     -- Frame 0 is not played twice in a row when the loop returns to the beginning.
```

Comment:
Really?  Why would you normally want to duplicate a frame?  Then you would have a weird pause in the middle of your animation.